lib/config/fileconf.go

```diff
@@ -2385,3 +2388,101 @@ type Okta struct {
 	// APITokenPath is the path to the Okta API token.
 	APITokenPath string `yaml:"api_token_path,omitempty"`
 }
+
+// JamfService is the yaml representation of jamf_service.
+// Corresponds to [types.JamfSpecV1].
+type JamfService struct {
+	Service `yaml:",inline"`
+	// Name is the name of the sync device source.
+	Name string `yaml:"name,omitempty"`
+	// SyncDelay is the initial sync delay.
+	// Zero means "server default", negative means "immediate".
+	SyncDelay *time.Duration `yaml:"sync_delay,omitempty"`
```

Comment:
Do the durations need to be pointers?

Reply:
The idea was to be able to tell apart set from unset, but I suppose that doesn't make a difference anymore. Done.